variables.tf

```
@@ -8,12 +8,28 @@ variable "enforce_domain_name" {
 
 variable "enforce_group_id" {
   type        = string
-  description = "Enforce IAM group ID to bind your AWS account to"
+  description = "DEPRECATED: Please use 'enforce_group_ids'. Enforce IAM group ID to bind your AWS account to"
   sensitive   = false
   nullable    = false
 
   validation {
     condition     = length(regexall("^[a-f0-9]{40}(\\/[a-f0-9]{16})*$", var.enforce_group_id)) == 1
```

As this is now optional, we need to relax the validation condition I think.

```suggestion
    condition     = var.enforce_group_id != "" : length(regexall("^[a-f0-9]{40}(\\/[a-f0-9]{16})*$", var.enforce_group_id)) == 1 : true
```